views/soon.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+</head>
+<body>
+  <nav>
+    <a href="./index.html">Home</a>
+    <a href="./projects.html">Projects</a>
+    <a href="./about.html">About</a>
+    <a href="./soon.html">Coming Soon</a>
+  </nav>
+  <h1> Coming Soon...</h1>
+  <img src="sooncat.jpg" width="275px" alt = "soon cat"/>
```

Comment:
Even though Ruby standards say that we should put spaces on either side of our `=`, in HTML we don't want to do that